### PR TITLE
Add more Basic/Electric Machine pages

### PR DESCRIPTION
**Pull request by svr333**
Depending whether I feel like it I might do some more later.

**Added:**
- Carbon Press
- Automatic Ignition Chamber
- Composter
- Crop Growth Accelerator
- Crucible
- Industrial Miners
- EXP-Collector
- Fluid Pump
- Food Composter
- Freezer
- Juicer
- Magnesium-powered Generator

**Changed:**
- Rename Automated Ignition Chamber to Automatic Ignition Chamber
- Rename Digital Miners to Industrial Miners
- Add 2 newlines on some random pages I crossed
- Fix image backup name on Ore Washer page
- Rename all instance of Magnesium Generator to Magnesium-powered Generator

### CODE REVIEW

File: pages/EXP-Collector.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+The EXP-Collector is an [Electric Machine](https://github.com/Slimefun/Slimefun4/wiki/Electric-Machines) which can be used to collect exp.
```
Comment (TheBusyBiscuit) [inline pages/EXP-Collector.md:1]: Should also mention how, like the fact that it stores it in Flasks of Knowledge.

File: pages/Industrial-Miners.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+The Industrial Miners are part of the [Basic Machines](https://github.com/Slimefun/Slimefun4/wiki/Basic-Machines) category.  
+
+These machines will dig up ores from the area underneath them and put them in a chest.
```
Comment (TheBusyBiscuit) [inline pages/Industrial-Miners.md:3]: *till bedrock

File: pages/Industrial-Miners.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+The Industrial Miners are part of the [Basic Machines](https://github.com/Slimefun/Slimefun4/wiki/Basic-Machines) category.  
+
+These machines will dig up ores from the area underneath them and put them in a chest.  
+You have to put fuel in the chest before it is able to start working, more info on each fuel can be found in the Slimefun guide.  
+
+![Industrial Miners setup](https://raw.githubusercontent.com/Slimefun/Slimefun4-Wiki/master/images/multiblock-industrial-miners.png)
+
+On the image you can see the setup of the Industrial Miner and the Advanced Industrial Miner respectively.
+
```
Comment (TheBusyBiscuit) [inline pages/Industrial-Miners.md:9]: A GIF of their animation would be super cool but it isn't necessary :o

**Review by TheBusyBiscuit — CHANGES_REQUESTED** [review]

File: pages/Automatic-Ignition-Chamber.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+The Automatic Ignition Chamber is part of the [Basic Machines](https://github.com/Slimefun/Slimefun4/wiki/Basic-Machines) category.  
+
+## Usage
+The Automatic Ignition Chamber can only be used as an extension on the smeltery.
```
Comment (Sfiguz7) [inline pages/Automatic-Ignition-Chamber.md:4]:
```suggestion
The Automatic Ignition Chamber can only be used as an extension on the Smeltery (and not the Makeshift Smeltery).  
```
People are dumb (but don't tell them), best to specify

**Review by Sfiguz7 — CHANGES_REQUESTED** [review]

File: pages/Industrial-Miners.md (hunk 3 above)
Comment (svr333) [reply pages/Industrial-Miners.md:9]: maybe some other time if thats okay

File: pages/Animal-Growth-Accelerator.md [hunk 5]
```diff
@@ -7,4 +7,4 @@ It runs on [Organic Food](https://github.com/Slimefun/Slimefun4/wiki/Miscellaneo
 There is only one tier of the Animal Growth Accelerator and it has the following properties:
```
Comment (Phoenix-Starlight) [inline pages/Animal-Growth-Accelerator.md:7]: What does it exactly do? Provide a description of what it does.

**Review by Phoenix-Starlight — DISMISSED** [review]
LGTM, except for one file which I have commented on.

File: pages/Animal-Growth-Accelerator.md (hunk 5 above)
Comment (svr333) [reply pages/Animal-Growth-Accelerator.md:7]: There is a description provided at the top line of this file
> The Animal Growth Accelerator is an [Electric Machine](https://github.com/Slimefun/Slimefun4/wiki/Electric-Machines) which can be used to speed up the growth of baby animals.

**Review by Phoenix-Starlight — APPROVED** [review]
Also forgot to approve.

**Review by Sfiguz7 — APPROVED** [review]
LGTM

**Review by TheBusyBiscuit — APPROVED** [review]

**Pull request merged**